platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_structuring_builds.adoc

```diff
@@ -592,3 +592,71 @@ You can also now invoke tasks on the `my-web-module` project using the shorter n
 
 === Tags
 `<<tags_reference.adoc#tag:structuring-builds,#structuring-builds>>`
+
+
+[[use_convention_plugins]]
+== Use Convention Plugins for Common Build Logic
+
+Use <<implementing_gradle_plugins_precompiled.adoc#implemention_precompiled_plugins,convention plugins>> to encapsulate and reuse shared build logic across multiple projects in your build.
+
+=== Explanation
+
+Instead of duplicating configuration across multiple buildscripts, you can easily move common logic into a reusable convention plugin.
```

Comment:
```suggestion
Instead of duplicating configuration across multiple build scripts, you can easily move common logic into a reusable convention plugin.
```